app/models/service_ansible_playbook.rb

```diff
@@ -53,7 +53,12 @@ def manageiq_extra_vars
       'api_token' => Api::UserTokenService.new.generate_token(evm_owner.userid, 'api'),
       'service'   => href_slug,
       'user'      => evm_owner.href_slug
-    }
+    }.merge(request_options_extra_vars)
+  end
+
+  def request_options_extra_vars
+    ro = miq_request_task.options[:request_options] || {}
```

Comment:
`miq_request_task.options.fetch_path(:request_options, :extra_vars) || {}`.
`:extra_vars` needs to be replaced with `:manageiq_extra_vars` or `:extra_vars, 'manageiq'` if we agree with the other comment.